src/app/shared/directives/border-highliter.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({
+  selector: '[crsBorderHighliter]'
+})
+export class BorderHighliterDirective {
```

Comment:
We have a typo in the name, name it: `HighlightBorderDirective` instead.

Reply:
Done